gsd/fl.pyx

```diff
@@ -163,53 +164,66 @@ def open(name, mode, application=None, schema=None, schema_version=None):
     |                  | Raise :py:exc:`FileExistsError`             |
     |                  | if it already exists.                       |
     +------------------+---------------------------------------------+
-    | ``'ab'``         | Open an existing file for reading and       |
-    |                  | writing. Does *not* create or overwrite     |
-    |                  | existing files.                             |
+    | ``'a'``          | Open a file for reading and writing.        |
+    |                  | Creates the file if it doesn't exist.       |
     +------------------+---------------------------------------------+
 
-    When opening a file for reading (``'r'`` and ``'a'`` modes): ``application``
-    and ``schema_version`` are ignored and may be ``None``. When ``schema`` is
-    not ``None``, :py:func:`open` throws an exception if the file's schema does
-    not match ``schema``.
-
-    When opening a file for writing (``'w'`` or ``'x'`` modes): The given
-    ``application``, ``schema``, and ``schema_version`` are saved in the file
-    and must not be None.
+    When opening a file for reading (``'r'`` and ``'r+'`` modes):
+    ``application`` and ``schema_version`` are ignored and may be ``None``.
+    When ``schema`` is not ``None``, :py:func:`open` throws an exception if the
+    file's schema does not match ``schema``.
+
+    When opening a file for writing (``'w'``, ``'x'``, or ``'a'`` modes): The
+    given ``application``, ``schema``, and ``schema_version`` must not be None.
+
+    .. deprecated:: 2.9.0
+
+        The following values to ``mode`` are deprecated:
+
+        +------------------+---------------------------------------------+
+        | mode             | description                                 |
+        +==================+=============================================+
+        | ``'rb'``         | Equivalent to ``'r'``                       |
+        +------------------+---------------------------------------------+
+        | ``'rb+'``        | Equivalent to ``'r+'``                      |
+        +------------------+---------------------------------------------+
+        | ``'wb'``         | Equivalent to ``'w'``                       |
+        +------------------+---------------------------------------------+
+        | ``'wb+'``        | Equivalent to ``'w'``                       |
+        +------------------+---------------------------------------------+
+        | ``'xb'``         | Equivalent to ``'x'``                       |
+        +------------------+---------------------------------------------+
+        | ``'xb+'``        | Equivalent to ``'x'``                       |
+        +------------------+---------------------------------------------+
+        | ``'ab'``         | Open an existing file for reading and       |
+        |                  | writing. Does *not* create or overwrite     |
+        |                  | existing files.                             |
```

Comment:
`ab` can be mapped to `r+` in all circumstances, right?

Reply:
Yes, you are correct.